lib/resources/r4/related_person_patch.yaml

```diff
@@ -89,9 +98,39 @@ operations:
       <ul>
         <li>The <code>value.coding</code> must contain exactly one proprietary system.</li>
         <li>The <code>value.coding</code> cannot contain a standard system.</li>
+        <li>When specifying a `period`, the fields must include a time component with a timezone.</li>
         <li>Duplicate relationships may be added, but will only be represented once on the Resource.</li>
       </ul>
 
+  - name: test-relationship-id
+    path: /relationship/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the `Relationship.id` value of the relationship at the given <code>{index}</code> in the list of related person relationships.
+    example: |
+      {
+        "op": "test",
+        "path": "/relationship/0/id",
+        "value": "CI-98278577-0"
+      }
+
+  - name: remove-relationship
+    path: /relationship/{index}
+    operation: remove
+    type: N/A
+    url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson.relationship
+    description: Remove the relationship at the given <code>{index}</code> in the list of related person relationships.
+    example: |
+      {
+        "op": "remove",
+        "path": "/relationship/0"
+      }
+    note: >
+      <ul>
+        <li>Requires a test operation to be provided for the Relationship.id field of the relationship to be removed.</li>
```

Comment:
We should probably make this 'Relationship.id' so that it falls in line with similar patch notes.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/650/commits/ee556c67d77937a3ff60d68720277fd3f1537305#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR131

Reply:
thank you.